docs/sources/clients/promtail/stages/geoip.md

```diff
@@ -0,0 +1,129 @@
+---
+title: geoip
+description: geoip stage
+---
+# geoip
+
+The `geoip` stage is a parsing stage that reads a ip address and 
+populates the labelset with geoip fields. Maxmind's GeoIP2 databse is used for the lookup.
+
+Populated fields for City db:
+
+- geoip_city_name
+- geoip_country_name
+- geoip_continet_name
+- geoip_continent_code
+- geoip_location_latitude
+- geoip_location_longitude
+- geoip_postal_code
+- geoip_timezone
+- geoip_subdivision_name
+- geoip_subdivision_code
+
+Populated fields for ASN db:
```

Comment:
And a third database mention.  Also, please spell out acronyms (Acronym Finder has 60+ entries for ASN).

Reply:
ASN is `Autonomous System Number`. Since, we are speaking of IPs here, it's more obvious. So, I feel it's okay not to expand it.

Reply:
As a rule, always spell out acronyms.  Don't assume that just because you know what the acronym means that others will also know. Spelling out acronyms is helpful for non-native English speakers, and absolutely essential for translation/localization purposes.